AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -93,6 +123,37 @@ def __init__(self, time_name: str, word_vector: WordVector) -> None:
         self.time_name: str = time_name
         self.word_vector: WordVector = word_vector
 
+    def convert_from_dict(
+        time_name: str, word_weights: Dict[str, float]
+    ) -> TimeFrame:
+        """
+        Convert from a dictionary of word and weight to TimeFrame instance.
+
+        This is static conversion method.
+
+        :param str time_name: Name of the time
+        :param Dict[str, float] word_weights: The words and their weights
+        :return: The TimeFrame instance
+        :rtype: TimeFrame
+        """
+
+        word_vector = WordVector.convert_from_dict(word_weights)
```

Comment:
word_vector = WordVector().convert_from_dict(word_weights)

Reply:
If "word_vector = WordVector.convert_from_dict(word_weights)" works well, you need not to change this line.